### PR TITLE
Delete dev env wf

### PR DESCRIPTION
**Pull request by ben851**
# Summary | Résumé

I've added in the logic to kill s3 buckets manually so that aws nuke doesn't choke

## Related Issues | Cartes liées

* https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/352

## Before merging this PR

Read code suggestions left by the
[cds-ai-codereviewer](https://github.com/cds-snc/cds-ai-codereviewer/) bot. Address
valid suggestions and shortly write down reasons to not address others. To help
with the classification of the comments, please use these reactions on each of the
comments made by the AI review:

| Classification      | Reaction | Emoticon |
|---------------------|----------|----------|
| Useful              | +1       | 👍        |
| Noisy               | eyes     | 👀        |
| Hallucination       | confused | 😕        |
| Wrong but teachable | rocket   | 🚀        |
| Wrong and incorrect | -1       | 👎        |

The classifications will be extracted and summarized into an analysis of how helpful
or not the AI code review really is.

## Test instructions | Instructions pour tester la modification

After a create dev env, try and run this.

## Release Instructions | Instructions pour le déploiement

None.

## Reviewer checklist | Liste de vérification du réviseur

* [ ] This PR does not break existing functionality.
* [ ] This PR does not violate GCNotify's privacy policies.
* [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
* [ ] This PR does not significantly alter performance.
* [ ] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: scripts/deleteEnvironment.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ fi
 
 # Set the account id in AWS Nuke config
 echo "Configuring AWS Nuke"
-sed -i 's/SCRATCH_ACCOUNT_ID/$ACCOUNT_ID/g' scripts/awsNuke.cfg
+sed -i'' -e "s/SCRATCH_ACCOUNT_ID/$ACCOUNT_ID/g" awsNuke.cfg
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/deleteEnvironment.sh:26]: This works for both linux and macos so I don't want to mess with it

File: scripts/deleteEnvironment.sh [hunk 2]
```diff
@@ -52,6 +52,14 @@ terragrunt destroy -var-file ../$ENVIRONMENT.tfvars --terragrunt-non-interactive
 popd
 echo "Done."
 
+# Delete Cloud Based Sensor Bucket
+echo "Deleting Cloud Based Sensor S3 Bucket..."
+pushd ../env/$ENVIRONMENT/common
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/deleteEnvironment.sh:57]: Popd is below!

File: scripts/deleteEnvironment.sh [hunk 3]
```diff
@@ -52,6 +52,14 @@ terragrunt destroy -var-file ../$ENVIRONMENT.tfvars --terragrunt-non-interactive
...
+terragrunt destroy -var-file ../$ENVIRONMENT.tfvars --target module.cbs_logs_bucket --terragrunt-non-interactive -auto-approve
+popd
+echo "Done."
+
+
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/deleteEnvironment.sh:62]: I don't see this locally. Weird.

File: scripts/deleteS3.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import boto3
+
+# PROFILE = "dev"
```
Comment (sastels) [inline scripts/deleteS3.py:3]: Did you mean to leave the comment in?

**Review by sastels — APPROVED** [review]
I think this looks good!

File: scripts/deleteS3.py (hunk 4 above)
Comment (ben851) [reply scripts/deleteS3.py:3]: I did not!

**Pull request merged**